src/Bedrock.Framework.Experimental/Protocols/Memcached/Serializers/ISerializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Text;
+
+namespace Bedrock.Framework.Experimental.Protocols.Memcached.Serializers
```

Comment:
This shoudn't be part of this API. The API should just support bytes and this is something that can be layered on top.

Reply:
Ok i undertand. Changes commited [https://github.com/davidfowl/BedrockFramework/pull/84/commits/0b9498e40f3bc3eac1df5d0e5178c1f521b1fac8](https://github.com/davidfowl/BedrockFramework/pull/84/commits/0b9498e40f3bc3eac1df5d0e5178c1f521b1fac8)